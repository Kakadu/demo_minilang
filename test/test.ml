
let () =
  match [%getenv "PATH"] with
  | None -> assert false
  | Some _ -> ()

let () = assert ([%getenv "DOES_NOT_EXIST"] = None)
