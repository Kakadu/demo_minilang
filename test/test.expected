let () =
  match match Sys.getenv "PATH" with
        | s -> Some s
        | exception Not_found  -> None
  with
  | None  -> assert false
  | Some _ -> () 
let () =
  assert
    ((match Sys.getenv "DOES_NOT_EXIST" with
      | s -> Some s
      | exception Not_found  -> None) = None)
  
